packages/trader/src/Modules/Trading/Containers/purchase.jsx

```diff
@@ -51,32 +53,48 @@ const Purchase = observer(({ is_market_closed }) => {
         const is_proposal_error =
             is_multiplier || (is_accumulator && !is_mobile) ? info.has_error && !!info.message : info.has_error;
         const purchase_fieldset = (
-            <PurchaseFieldset
-                basis={basis}
-                buy_info={purchase_info}
-                currency={currency}
-                growth_rate={growth_rate}
-                info={info}
-                key={index}
-                index={getSortedIndex(index, type)}
-                has_cancellation={has_cancellation}
-                is_accumulator={is_accumulator}
-                is_disabled={is_disabled}
-                is_high_low={is_high_low}
-                is_loading={isLoading(info)}
-                is_market_closed={is_market_closed}
-                is_mobile={is_mobile}
-                is_multiplier={is_multiplier}
-                is_turbos={is_turbos}
-                is_vanilla={is_vanilla}
-                is_proposal_empty={is_proposal_empty}
-                is_proposal_error={is_proposal_error}
-                purchased_states_arr={purchased_states_arr}
-                onHoverPurchase={onHoverPurchase}
-                onClickPurchase={onClickPurchase}
-                setPurchaseState={setPurchaseState}
-                type={type}
-            />
+            <div className='trade-params--mobile__payout-container'>
```

Comment:
Please, make sure that this new style want break anything for Vanillas + all the rest contracts